integreat_compass/cms/fixtures/test_data.json

```diff
@@ -248,24 +248,21 @@
     "model": "auth.group",
     "pk": 1,
     "fields": {
-      "name": "BOARD_MEMBER",
-      "permissions": [38, 37]
+      "name": "BOARD_MEMBER"
```

Comment:
The permissions fixture also contains the groups itself, right? Maybe we could remove it here completely, so we don't get the wrong permissions when the fixtures are imported in the wrong order?

Reply:
I think the correct logic would be to load the groups & permissions first, and then the rest of the test data (without any group information) afterwards.

Reply:
Alright